app/services/UserService.py

```diff
@@ -31,3 +31,20 @@ async def modify_user(user_id: int, model: PydanticUserModify):
 
     except ValueError as e:
         raise HTTPException(status_code=409, detail=str(e)) from e
+
+async def get_all_users() -> list[PydanticUserResponse]:
+    """
+    Retrieves all users.
+    """
+    users = await UserInDB.all()
+    return [PydanticUserResponse.from_orm(user) for user in users]  # Use from_orm for each user
+
+async def get_user_by_id(user_id: int) -> PydanticUserResponse:
+    """
+    Retrieves a user by their ID.
+    """
+    user = await UserInDB.get_or_none(id=user_id)
+    if user is None:
+        raise HTTPException(status_code=404, detail=CommonErrorMessages.USER_NOT_FOUND)
+    
+    return PydanticUserResponse.from_orm(user)  # Use from_orm to create the response model
```

Comment:
from_orm is depreciated since Pydantic V2.
Please use model_validate instead.

Reply:
done